taxi/views.py

```diff
@@ -44,6 +66,26 @@ class CarDetailView(LoginRequiredMixin, generic.DetailView):
     model = Car
 
 
+class CarCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Car
+    success_url = reverse_lazy("taxi:car-list")
+    fields = ["model", "manufacturer", "drivers"]
+    template_name = "taxi/car_form.html"
+
+
+class CarUpdateView(LoginRequiredMixin, UpdateView):
+    model = Car
+    success_url = reverse_lazy("taxi:car-list")
+    fields = ["model", "manufacturer", "drivers"]
+    template_name = "taxi/car_form.html"
+
+
+class CarDeleteView(LoginRequiredMixin, DeleteView):
+    model = Car
+    success_url = reverse_lazy("taxi:car-list")
+    template_name = "taxi/car_confirm_delete.html"
```

Comment:
Code Efficiency
Don't override template_name, context_object_name and so on if they are the same as default ones.